apps/omg/test/omg/raw_data_test.exs

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.RawDataTest do
+  use ExUnit.Case, async: true
+  # doctest OMG.RawData
+  alias OMG.RawData
+
+  test "parsing amounts" do
+    big_32bytes = 2.0 |> :math.pow(8 * 32) |> Kernel.trunc()
+    big_just_enough = big_32bytes - 1
+
+    rlp_data = ExRLP.encode([0, 1, big_just_enough, big_32bytes])
+    [zero, one, big, too_big] = ExRLP.decode(rlp_data)
+
+    assert {:error, :amount_cant_be_zero} == RawData.parse_amount(zero)
```

Comment:
I agree the only concern is to not contain exact error code in the test name - in case we ever needed to change it, the test name will probably will be left omitted.